### DISCUSSION TITLE
Push to GitHub is greyed out

### DISCUSSION THREAD

**Discussion by saerts** [six7/figma-tokens#1716] [General]
I'm connected with GitHub, I can pull the tokens. But I'm not able to change the tokens. When I create a theme I get a blue dot to push to GitHub, but greyed out ??

**Comment by StefanIvanov** [comment]
I had same issue yesterday and figured out it was due to the PAT and the way it was set up in Github. Can you check if it is setup with everything under repo checked?
![Screenshot 2023-03-22 at 16 24 56](https://user-images.githubusercontent.com/1702064/226934882-34fd4e23-8c1c-4bb3-b034-0e2a6d0b6789.png)
You may have to generate a new token, use the classic way to do that.

**Reply by saerts** [reply]
That was indeed the issue
